lib/__tests__/standalone-cache.test.js

```diff
@@ -196,3 +196,24 @@ describe('standalone cache uses cacheLocation', () => {
 		expect(cache.getKey(validFile)).toBeTruthy();
 	});
 });
+
+describe('standalone cache uses cacheStrategy', () => {
+	const expectedCacheFilePath = path.join(cwd, '.stylelintcache');
+
+	afterEach(async () => {
+		// clean up after each test
+		await removeFile(expectedCacheFilePath);
+	});
+
+	it('cacheStrategy is "contenet"', async () => {
+		await standalone(getConfig({ cacheStrategy: 'content' }));
+
+		// Ensure cache file is created
+		expect(existsSync(expectedCacheFilePath)).toBe(true);
+
+		const { cache } = fCache.createFromFile(expectedCacheFilePath, undefined, true);
+
+		expect(typeof cache.getKey(validFile)).toBe('object');
+		expect(cache.getKey(validFile)).toBeTruthy();
+	});
```

Comment:
[suggest] I think we'd like to test the behavior at a higher level instead of using `fCache`. For example,

```js
const cacheStrategy = 'content';

await fs.copyFile(validFile, newFileDest);

const { results } = await standalone(getConfig({ cacheStrategy }));
expect(results.some((file) => file.source === newFileDest)).toBe(true);

// No content change, but file metadata id changed
await fs.utimes(newFileDest, Date.now(), Date.now());

const { results: resultsCached } = await standalone(getConfig({ cacheStrategy }));
expect(resultsCached.some((file) => file.source === newFileDest)).toBe(false);
```

I guess it's better also to add a test case for `cacheStrategy = 'metadata'` since we can compare the `content` case easily.

What do you think?

Reply:
This test case looks better than the code I wrote.

https://github.com/stylelint/stylelint/pull/6357/commits/5b5088f043441398bdbdfea76897f9207b5d824d